asyn/drvAsynSerial/drvAsynIPPort.c

```diff
@@ -520,6 +532,9 @@ connectIt(void *drvPvt, asynUser *pasynUser)
                     tty->flags |=  FLAG_NEED_LOOKUP;
                 return asynError;
             }
+            if (setsockopt(fd, SOL_SOCKET, SO_SNDTIMEO, (char *)&saveTV, sizeof saveTV) < 0) {
+                asynPrint(pasynUser, ASYN_TRACE_ERROR, "connectIt, error calling setsockopt for SO_RECVTIMEO: %s\n", strerror(SOCKERRNO));
```

Comment:
All 3 error messages are using the wrong option name (RECV instead of SND).

Reply:
I fixed that, thanks for catching it.